src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2294,12 +2294,19 @@ private function resolveTrustedHeaders(array $headers): int
                 case 'x-forwarded-host': $trustedHeaders |= Request::HEADER_X_FORWARDED_HOST; break;
                 case 'x-forwarded-proto': $trustedHeaders |= Request::HEADER_X_FORWARDED_PROTO; break;
                 case 'x-forwarded-port': $trustedHeaders |= Request::HEADER_X_FORWARDED_PORT; break;
-                case '!x-forwarded-host': $trustedHeaders &= ~Request::HEADER_X_FORWARDED_HOST; break;
+                case '!x-forwarded-host':
```

Comment:
I forgot that this has been introduced in 5.2.
Let's drop it from 5.2 actually! (and maybe target this PR on 5.2 instead of 5.3 then!)

Reply:
agreed. Creating the deprecation of a feature before releasing it does not make sense.